envolve-backend/src/main/java/de/neuefische/erbay/envolve/model/Teacher.java

```diff
@@ -10,12 +10,12 @@
 @Data
 @AllArgsConstructor
 @NoArgsConstructor
-@Document(collection = "user")
+@Document(collection = "teacher")
```

Comment:
Maybe it would be better to have a use with a role teacher. Otherwise you could get a problem when other users maybe need a login 🤔

Reply:
In the branch backendsetup its already "teacher" and "student" in the models :)

Reply:
Or do you mean instead of teacher/student better just one user collection?